docs/skopeo-standalone-sign.1.md

```diff
@@ -25,6 +25,10 @@ $ skopeo standalone-sign busybox-manifest.json registry.example.com/example/busy
 $
 ```
 
+## NOTES
+
+This command is intended for use with local signatures, such as OpenPGP, as per containers-signature(5). Furthermore, this command does **not** interact with the artifacts generated by Docker Content Trust (DCT). For more information, please see [containers-signature(5)](https://github.com/containers/image/blob/main/docs/containers-signature.5.md).
```

Comment:
I don’t care very much at all whether this links to containers-signature(5) the man page, that _and_ a HTTP link, or neither (@rhatdan which one do you prefer?), but it would be neat and tidy for the two standalone-* man pages to have the same text in the NOTES section.

Reply:
Don't mind either way, I'll go with the consensus of you @mtrmac and @rhatdan 🙇‍♀️

Reply:
OK, made another commit @mtrmac - is this a little better ? 🙏

Reply:
No response from Dan, so let’s finally get this out of the door.